firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallations.java

```diff
@@ -54,6 +55,8 @@
   private final Clock clock;
   private final Utils utils;
 
+  private static final long TOKEN_EXPIRATION_BUFFER = 3600L;
```

Comment:
what does this variable represent? is it a time period?

Reply:
Yes it is. 1 hour buffer time to evaluate if the auth token has expired.

Reply:
Mention the time unit in the var name? for example secs

Reply:
Done.